MonoGame.Framework.Content.Pipeline/ExternalTool.cs

```diff
@@ -168,8 +168,7 @@ private static void EnsureExecutable(string path)
 
             try
             {
-
-                var p = Process.Start("chmod", "u+x '" + path + "'");
+                var p = Process.Start("chmod", "u+x " + path);
```

Comment:
If path contains spaces this will fail.

Reply:
Hmm, I don't remember why I did that. Will fix!

Reply:
Fixed!